volttron/platform/control.py

```diff
@@ -934,6 +934,47 @@ def update_auth(opts):
         _stderr.write('ERROR: %s\n' % err.message)
 
 
+def _show_filtered_agents(opts, field_name, field_callback, agents=None):
```

Comment:
Please document what inputs and outputs of this function are...and why this is necessary.